tests/components/geonetnz_quakes/test_geo_location.py

```diff
@@ -151,6 +154,7 @@
 
         all_states = hass.states.async_all()
         assert len(all_states) == 1
+        assert len(entity_registry.entities) == 1
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5122969)